.github/workflows/jrgitlint.yml

```diff
@@ -0,0 +1,19 @@
+name: JR GitLint
+
+on: [push, pull_request]
+
+jobs:                                                                            
+  gitlint:                                                                       
+                                                                                 
+    runs-on: ubuntu-latest                                                       
+                                                                                 
+    container:                                                                   
+      image: jorisroovers/gitlint                                                
+                                                                                 
+    steps:                                                                       
+    - uses: actions/checkout@v2                                                  
+    - name: git fetch                                                              
+      run: git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
```

Comment:
ah, I don't think is needed, the `/action/checkout above should have gotten the code. And it would only checkout 1 commit depth anyway.
I think you just need to use `run: gitlint --commits ${GITHUB_BASE_REF}..${GITHUB_HEAD_REF}`